packages/core/src/extensions/Blocks/nodes/BlockContent/ListItemBlockContent/NumberedListItemBlockContent/NumberedListItemBlockContent.ts

```diff
@@ -65,18 +66,35 @@ export const NumberedListItemBlockContent = Node.create({
             return false;
           }
 
-          // Case for BlockNote list structure.
-          if (parent.getAttribute("data-content-type") === "numberedListItem") {
+          if (parent.tagName === "OL") {
             return {};
           }
 
-          // Case for regular HTML list structure.
-          if (parent.tagName === "OL") {
+          return false;
+        },
+        node: "blockContainer",
+      },
+      // Case for BlockNote list structure.
```

Comment:
How does this work again / why is this used?

Reply:
It's just so that ordered/unordered list items are parsed correctly from HTML. Since BlockNote doesn't use `ol`/`ul` tags to represent lists, we need 2 separate HTML parse rules - one for list items coming from BlockNote, and the other for list items coming from everywhere else. This was changed a bit since I changed BlockNote list items to have `p` tags instead of `li` (having `li` elements without a parent `ol`/`ul` element isn't valid HTML).